README.md

```diff
@@ -84,8 +87,12 @@ to find out the default location.
 Create or download an fv3config yaml configuration. Edit the configuration as needed.
 Examples of such configurations are included in the tests under `tests/pytest/config`.
 
-Once you have a configuration file, e.g. `example/config.yml`, you can write a run directory in python using:
+Once you have a configuration file, e.g. `example/config.yml`, you can write a run directory as follows:
+
+    write_run_directory example/config.yml rundirectory
```

Comment:
This glosses over the fact that one requires a functioning Google cloud CLI tools setup and the fact that this fetches from a public request-pays bucket. This is actually where most people will stumble following these instructions.

Reply:
Great point about the requester pays bucket, I'll point to gcsfs's authentication instructions as well.

Reply:
Maybe the fv3config README already is a great resource? Haven't looked at those for some time...

Reply:
It doesn't look like the fv3config docs mention authentication.